### PR TITLE
add functionalities for new ore standard token

### PR DESCRIPTION
**Pull request by surabhil**
- Add a orestandardtoken.js with the ore.standard_token functionalities
- Update CPU functions according to ore.standard_token
- Update example account_create_random.js to be fully functional with new cpu contract

Note: This PR includes the work to implement only CPU and not ORE as ore.standard_token.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,6 +22,7 @@ class Orejs {
     Object.assign(this, eos)
     Object.assign(this, instrument)
     Object.assign(this, ore)
+    Object.assign(this, oreStandardToken)
```
Comment (blackwatertepes) [inline src/index.js]: How does the oreStandardToken differ from token? Should all of these methods be inside of the token module?

Comment (blackwatertepes) [reply src/index.js]: ;)

File: src/orestandardtoken.js [hunk 2]
```diff
@@ -19,12 +44,23 @@ async function getStandardTokenBalance(accountName, tokenSymbol, contractName) {
...
+  // Standard token transfer
   const { contract, options } = await this.contract(contractName, fromAccountName)
   await contract.transfer(fromAccountName, toAccountName, tokenAmount.toString(), memo, options);
 }
 
+// cleos push action cpu.ore transferfrom '["app.apim", "test1.apim", "test2.apim", "10.0000 CPU"]' -p app.apim
+async function transferfrom(approvedAccountName, fromAccountName, toAccountName, tokenAmount,contractName) {
+  // Standard token transfer
+  const { contract, options } = await this.contract(contractName, approvedAccountName)
+  await contract.transferfrom(approvedAccountName, fromAccountName, toAccountName, tokenAmount.toString(), options);
```
Comment (blackwatertepes) [inline src/orestandardtoken.js]: Why do we need to pass in the approved account? The approved account is the one making the request.

Comment (blackwatertepes) [reply src/orestandardtoken.js]: How is this different from the transfer method?

Comment (surabhil) [reply src/orestandardtoken.js]: Transfer method is the standard EOS transfer method and transferfrom is what we implemented  to have one account transfer on behalf of another.

**Pull request merged**